ui/src/Components/threadview/SearchBox.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import '../messageview/message.css';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import { TextField } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+const SearchBox = () => {
+    const useStyles = makeStyles(theme => ({
+        root: {
+            width: '100%',
+            maxWidth: '100%',
+            backgroundColor: theme.palette.background.paper,
+        },
+    }));
+    const classes = useStyles();
+    return (
+        <div>
+            <List className="send-container" dense={false}>
```

Comment:
List is not needed. A simple FormControl and a TextField is all you need.

Reply:
Changed to TextField